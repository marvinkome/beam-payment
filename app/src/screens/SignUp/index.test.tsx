import React from "react"
import { render } from "@testing-library/react-native"
import { SignUp } from "./index"

describe("SignUp page tests", () => {
    test("Renders main screen", () => {
        const renderedComp = render(<SignUp />)
        expect(renderedComp.toJSON()).toMatchSnapshot()
    })
})
