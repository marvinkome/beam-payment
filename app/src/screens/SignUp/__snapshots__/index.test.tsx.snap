// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp page tests Renders main screen 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "paddingHorizontal": 20,
      "paddingVertical": 15,
    }
  }
>
  <View>
    <View
      style={
        Object {
          "paddingVertical": 15,
        }
      }
    >
      <SvgMock
        width={100}
      />
    </View>
    <Text
      style={
        Object {
          "color": "#242424",
          "fontSize": 51.42857142857143,
          "letterSpacing": 0.8,
          "marginVertical": 45,
          "textAlign": "center",
        }
      }
    >
      Send money to anyone with a phone number
    </Text>
    <View
      style={
        Object {
          "paddingHorizontal": 10,
          "width": "100%",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "borderBottomWidth": 1,
            "borderColor": "#9175DB",
            "borderRadius": 50,
            "flexDirection": "row",
            "marginBottom": 50,
            "marginVertical": 20,
            "transform": Array [
              Object {
                "translateX": 0,
              },
            ],
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "height": 40,
              "justifyContent": "center",
              "marginVertical": 4,
              "paddingRight": 4,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
              }
            }
          >
            <SvgMock
              style={
                Object {
                  "marginLeft": 15,
                }
              }
              testID="countryIcon"
            />
            <Text
              style={
                Object {
                  "color": "#242424",
                  "marginLeft": 15,
                  "marginRight": 5,
                }
              }
            >
              +234
            </Text>
          </View>
        </View>
        <TextInput
          allowFontScaling={true}
          editable={true}
          keyboardType="number-pad"
          placeholder="Enter phone number"
          placeholderTextColor="#86939e"
          rejectResponderTermination={true}
          style={
            Object {
              "alignSelf": "center",
              "color": "#242424",
              "flex": 1,
              "fontSize": 18,
              "minHeight": 40,
              "paddingLeft": 0,
            }
          }
          testID="RNE__Input__text-input"
          underlineColorAndroid="transparent"
        />
      </View>
      <Text
        style={
          Object {
            "color": "#ff190c",
            "fontSize": 12,
            "margin": 5,
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        }
      }
    >
      <View
        accessibilityRole="button"
        accessibilityState={
          Object {
            "busy": false,
            "disabled": true,
          }
        }
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "hsl(208, 8%, 90%)",
              "borderColor": "#2089dc",
              "borderRadius": 3,
              "borderWidth": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "padding": 8,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "hsl(208, 8%, 63%)",
                "fontSize": 18,
                "paddingVertical": 1,
                "textAlign": "center",
              }
            }
          >
            Continue
          </Text>
        </View>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
